src/tracks/playabletrack/wavetrack/ui/WaveChannelView.cpp

```diff
@@ -848,17 +848,7 @@ void WaveChannelSubView::DrawBoldBoundaries(
       if (xx >= 0 && xx < rect.width) {
          dc.SetPen( highlightLoc ? AColor::uglyPen : *wxGREY_PEN );
          AColor::Line(dc, (int) (rect.x + xx - 1), rect.y, (int) (rect.x + xx - 1), rect.y + rect.height);
-         if (loc.typ == WaveTrackLocation::locationCutLine) {
-            dc.SetPen( highlightLoc ? AColor::uglyPen : *wxRED_PEN );
-         }
-         else {
-#ifdef EXPERIMENTAL_DA
-            // JKC Black does not show up enough.
-            dc.SetPen(highlightLoc ? AColor::uglyPen : *wxWHITE_PEN);
-#else
-            dc.SetPen(highlightLoc ? AColor::uglyPen : *wxBLACK_PEN);
-#endif
-         }
```

Comment:
Suppress drawing of clip edges only at close magnifications.

Reply:
I don't understand. The only remaining possibility is for `loc.typ` to equal `WaveTrackLocation::locationCutLine`, and the remaining code is what was executed in that situation.